__tests__/app.test.js

```diff
@@ -92,6 +92,54 @@ describe("/api/reviews/:review_id", () => {
       });
   });
 });
+describe.only("/api/reviews/:review_id/comments", () => {
+  test("GET - 200: Returns the review with the correct ID", () => {
+    return request(app)
+      .get("/api/reviews/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.length).toBe(3);
+        expect(body).toBeSortedBy("created_at", { descending: true });
+        body.forEach((comment) => {
+          expect(comment).toEqual(
+            expect.objectContaining({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              review_id: expect.any(Number),
+            })
+          );
+        });
+      });
+  });
+  test("GET - 200: Returns an empty array if a review has no comments", () => {
+    return request(app)
+      .get("/api/reviews/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.length).toBe(0);
+        expect(body).toEqual([]);
+      });
+  });
+  test("GET - 400: Bad request", () => {
+    return request(app)
+      .get("/api/reviews/dog/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+  test("GET - 400: Good request but id does not exist", () => {

```

Comment:
As this is a valid but non-existent ID, a 404 error would be better here as the client hasn't actually provided a bad request.